README.md

```diff
@@ -1,7 +1,190 @@
+
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```
+ {(a,b)(b,a)(a,a)} 
+ The second statement is true in this model. There is an x that relates every y (a, related to a and b) There is an x that relates to a z (where a is x and b is z), all ys (a and b) are related to some z (a is related to a and b, b is related to a)
+ The first statement is not true. It is not true that all xs (a and b) are related to all zs (b is not related to a)
+A graph is provided in the attached file.
+
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+  ```
+See attached images.
+
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+```
+	
+9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+
+Using http://teachinglogic.liglab.fr/DN/index.php, since I couldn't make the other website work.
+
+a) 
+assume -(Px & Qx).
+  assume -(-Px + -Qx).
+    assume -Px.
+      -Px + -Qx.
+      F.
+    therefore --Px.
+    assume Qx.
+      Px.
+      assume -(-Px + -Qx).
+        Px & Qx.
+        F.
+      therefore --(-Px + -Qx).
+      F.
+    therefore -Qx.
+    -Px + -Qx.
+    F.
+  therefore --(-Px + -Qx).
+  -Px + -Qx.
+therefore -(Px & Qx) => -Px + -Qx.	
+
+b)
+assume -(Px + Qx).
+  assume Px.
+    Px + Qx.
+    F.
+  therefore -Px.
+  assume Qx.
+    Px + Qx.
+    F.
+  therefore -Qx.
+  -Px & -Qx.
+therefore -(Px + Qx) => -Px & -Qx.	    
+
+c)
+assume -Px + -Qx.
+  assume Px & Qx.
+    Px.
+    Qx.
+    assume -Px.
+      F.
+    therefore -Px => F.
+    assume -Qx.
+      F.
+    therefore -Qx => F.
+    F.
+  therefore -(Px & Qx).
+therefore -Px + -Qx => -(Px & Qx).
+
+d)
+assume -Px & -Qx.
+  assume Px + Qx.
+    assume Px.
+      -Px.
+      F.
+    therefore Px => F.
+    assume Qx.
+      -Qx.
+      F.
+    therefore Qx => F.
+    F.
+  therefore -(Px + Qx).
+therefore -Px & -Qx => -(Px + Qx).	    
+
+10. Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs. 
+ 
+The tree method starts by assuming the negation of the statement to prove. It then applies a set of rule of derivations in order to find closed branches which provide a countermodel to the negated statement and ultimately prove it wrong (thus proving the original statement).

```

Comment:
I suspect that tree proof has only one rule: indirect derivation. But I'm not sure.

Reply:
I am pretty sure you are right, trees can only prove by contradiction/by providing a countermodel as far as I know